.github/workflows/lint.yml

```diff
@@ -66,3 +66,13 @@ jobs:
         with:
           tool: taplo-cli
       - run: taplo fmt --check
+  outdated:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Nit: you could probably use `@v4` here.

Reply:
Let's update all of these at once: https://github.com/0xPolygonZero/zk_evm/issues/759